src/models/fhir/parse.ts

```diff
@@ -22,7 +22,7 @@ const fhir = new Fhir();
  * @param resource Raw FHIR resource
  * @returns Parsed FHIR resource (simplified)
  */
-const parsers = (resource: R4.IResourceList | undefined) => {
+export const parsers = (resource: R4.IResourceList | undefined) => {
```

Comment:
Is there a need to export `parsers()`? I think it's only used internally by `parse()`.

Reply:
oh.. sorry, i forgot to reply about this. @HJunyuan  Actually, i used this in cloudwatch middleware for parsing observation to grab test name to write log for metric.